packages/expo/DangerZone.js

```diff
@@ -1,2 +1,3 @@
 // A convenience module that allows writing "import { X } from 'expo/DangerZone'"
-export * from './build/Expo/DangerZone';
+import * as DangerZone from './build/DangerZone';
+export default DangerZone;
```

Comment:
It's not correct because `DangerZone` wasn't exported as `default`, so it won't be possible to do
`import { Screen } from 'expo/DangerZone';` so this file wouldn't make sense.

Since it's not typescripted we can just use `module.exports`, this should work as previously:
```
import DangerZone from './build/DangerZone';

module.exports = DangerZone;
```

Reply:
Oh yeah, I think you're right, I remember now that this worked and I probably committed wrong version to Git. 👍

Reply:
Haha, committed proper one, just didn't push. 😅